src/components/plots/plot.ts

```diff
@@ -142,9 +142,10 @@ module Plottable {
     }
 
     public attr<D>(attr: string): Plots.AccessorScaleBinding<D, number | string>;
-    public attr(attr: string, attrValue: number | string | _Accessor): Plot;
-    public attr<D>(attr: string, attrValue: D | _Accessor, scale: Scale<D, number | string>): Plot;
-    public attr<D>(attr: string, attrValue?: number | string | _Accessor | D, scale?: Scale<D, number | string>): any {
+    public attr(attr: string, attrValue: number | string | Accessor<number> | Accessor<string>): Plot;
+    public attr<D>(attr: string, attrValue: D | Accessor<D>, scale: Scale<D, number | string>): Plot;
+    public attr<D>(attr: string, attrValue?: number | string | Accessor<number> | Accessor<string> | D | Accessor<D>,
```

Comment:
Here you used `<D>` for the generic, presumably because `<D>` corresponds to "domain" on the `Scale`. Elsewhere, you used a letter that corresponded to the property being set, such as `sectorValue<S>`. Do we want to use `<A>` here instead, or is it different because it's `attr()` and not a property-setter?